src/main/resources/config.yml

```diff
@@ -13,3 +13,6 @@ siteTypeCategories:
   - FOOD
```

Comment:
just for fun, let's add minimally:
FINANCIAL:
    BANK
HEALTH
    HOSPITAL
    PHARMACY

remove FOOD and replace with both GROCERY and RESTAURANT please. just so we don't need to touch for a while

Reply:
Fixed